lib/ConstExtract/ConstExtract.cpp

```diff
@@ -207,33 +248,52 @@ gatherConstValuesForPrimary(const std::unordered_set<std::string> &Protocols,
   return Result;
 }
 
-std::string toString(const CompileTimeValue *Value) {
-  switch (Value->getKind()) {
-    case CompileTimeValue::RawLiteral:
-      return cast<RawLiteralValue>(Value)->getValue();
-    case CompileTimeValue::InitCall:
-      // TODO
-    case CompileTimeValue::Builder:
-      // TODO
-    case CompileTimeValue::Dictionary:
-      // TODO
-    case CompileTimeValue::Runtime:
-      return "Unknown";
+void writeProperty(llvm::json::OStream &JSON,
+                   const ConstValueTypePropertyInfo PropertyInfo) {
+  const auto *PropertyDecl = PropertyInfo.VarDecl;
+  JSON.attribute("label", PropertyDecl->getName().str().str());
+  JSON.attribute("type",
+                 toFullyQualifiedTypeNameString(PropertyDecl->getType()));
+  JSON.attribute("isStatic", PropertyDecl->isStatic() ? "true" : "false");
+  JSON.attribute("isComputed", !PropertyDecl->hasStorage() ? "true" : "false");
+
+  auto value = PropertyInfo.Value.get();
+  switch (value->getKind()) {
+  case CompileTimeValue::ValueKind::RawLiteral:
```

Comment:
I know `{ }` is only required by the language when the first statement of a `case` is a declaration (e.g. line 268), but I do favor consistent use of braces for all cases within this `switch`.

Reply:
Added.